raincloud/__init__.py

```diff
@@ -51,33 +51,33 @@ def chpasswd():
         # pass the new_pass as an argument to the change password script like this:
         # bash ~/project_rainstorm/scripts/pw_change.sh new_pass
         # else return an error message: wrong current_pass
-        return 
+        return
 
     # TODO: set the license key
     @app.route('/settings/activate', methods=['POST'])
     def setLicense():
         # when a user buys a license, they will get a key via email
         # they then activate the license by posting it here
         # this route will store the license in a file somewhere for validating premium actions
-        return 
+        return
 
     # TODO: enable backups on the device
     @app.route('/backups/enable', methods=['POST'])
     def enableBackups():
         # call the ~/project_rainstorm/scripts/backups_enable.sh script
-        return 
-    
```

Comment:
We should add a linter to automatically catch these issues

Reply:
Create issue to add prettier.io into our workflow